Orange/regression/linear.py

```diff
@@ -149,14 +163,3 @@ def __str__(self):
         return 'LinearModel {}'.format(self.skl_model)
 
 
-class PolynomialModel(Model):
-    def __init__(self, model, polyfeatures):
-        self.model = model
-        self.polyfeatures = polyfeatures
```

Comment:
The alternative to moving large blocks around, in this case, is patching the type afterwards. I.e., somewhere at the bottom:
```py
PolynomialLearner.__returns__ = PolynomialModel
```